src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -2641,7 +2639,7 @@
 	 * @return a transforming {@link Flux} that emits tuples of time elapsed in milliseconds and matching data
 	 */
 	public final Flux<Tuple2<Long, T>> elapsed() {
-		return compose(f -> f.map(new Elapsed<>()));
+		return transform(f -> f.map(new Elapsed<>()));
```

Comment:
`Elapsed` is [stateful](https://github.com/reactor/reactor-core/blob/master/src/main/java/reactor/core/publisher/Elapsed.java#L29) so it needs to be deferred for each subscriber.